### ISSUE TITLE
Narrator saying control type as header for ribbon tab

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#792]
Copied from VSO ID: 544458

Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes

 
 
 **Problem description:**
 Narrator saying control type as header for ribbon tab

 **Minimal repro:**
 Steps to reproduce:
Create a ribbon in wpf and add a ribbon tab with some header like

```
<Ribbon>
        <RibbonTab Header="Home">            
        </RibbonTab>
</Ribbon>
```

For above ribbon tab, when narrator focuses, it says its control type as header instead of tab (see attached screenshot).

![home-header](https://user-images.githubusercontent.com/49078227/58647248-f86e8a80-82bb-11e9-9ac9-63c70796f619.PNG)

**Initial Commentary from Team:**
For Ribbons, the header is what receives focus (RibbonTabs are by default not focusable) we attempted to reroute Automation focus to the RibbonTab so that it would get announced correctly, but this lead to strange behaviors when dealing with SelectionPattern.RemoveFromSelection. We may need to re-evaluate the way we expose Ribbon->RibbonTab->RibbonHeader to UIAutomation

**Comment by grubioe** [comment]
@arpitmathur - can you review and assess if its needed for 3.0?

**Comment by anjalisheel-wpf** [comment]
@grubioe Here the control used is `RibbonTabHeadersPanel`, it is of header type only, thats why its control type is `Header` only. This is working as per the design.
